src/ImageSharp/Memory/Allocators/UniformUnmanagedMemoryPoolMemoryAllocator.cs

```diff
@@ -74,6 +74,10 @@ public UniformUnmanagedMemoryPoolMemoryAllocator(int? maxPoolSizeMegabytes)
             this.nonPoolAllocator = new UnmanagedMemoryAllocator(unmanagedBufferSizeInBytes);
         }
 
+        // This delegate allows overriding the method returning the available system memory,
+        // so we can test our workaround for https://github.com/dotnet/runtime/issues/65466
+        internal static Func<long> GetTotalAvailableMemoryBytes { get; set; } = () => GC.GetGCMemoryInfo().TotalAvailableMemoryBytes;
```

Comment:
You're missing a condition here.

#if NETCOREAPP3_1_OR_GREATER

Reply:
@antonfirsov I've pushed the fix.

Reply:
I missed the comment and also pushed it 🙈

Reply:
Doubled fixed now then. We know it's good 😄